src/DeviceInterface/InterfacePrototype.js

```diff
@@ -492,11 +496,13 @@ class InterfacePrototype {
     }
 
     async setupAutofill () {}
-    getAddresses () {}
-    /**
-     * @returns {Promise<null|Record<any,any>>}
-     */
+
+    /** @returns {Promise<EmailAddresses>} */
+    async getAddresses () { throw new Error('unimplemented') }
```

Comment:
This will now throw in Android, is this not an issue?

Reply:
https://github.com/duckduckgo/duckduckgo-autofill/blob/68aa923a71940e3d2acfb7743465c2ebe2da78e4/src/DeviceInterface/InterfacePrototype.js#L320-L326 covers this, we should clean up these code paths to be a little simpler to follow.